lte/gateway/c/core/oai/CMakeLists.txt

```diff
@@ -58,19 +58,6 @@ set(CMAKE_EXE_LINKER_FLAGS_DEBUG  "${CMAKE_EXE_LINKER_FLAGS_DEBUG} -fprofile-arc
 set(CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS_RELWITHDEBINFO} -fsanitize=leak -fno-omit-frame-pointer")
 set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO  "${CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO} -fsanitize=leak")
 
-################################################################
-# Add a flag to enable sentry if the library exists
-################################################################
```

Comment:
Can we keep this? (Or did it move to some other file?)

Reply:
rebase issue I think. Let me take a pass at this file again.